cwl/tools/annoSV.cwl

```diff
@@ -0,0 +1,26 @@
+class: CommandLineTool
+cwlVersion: v1.0
+id: annoSV
+requirements:
+  - class: ShellCommandRequirement
+  - class: ResourceRequirement
+    ramMin: 1000
+  - class: DockerRequirement
+    dockerPull: 'gaonkark/annotsv:latest'
```

Comment:
If you don't pull often, this should be ok. Otherwise, might be good to push this docker to SBG dockerhub. Otherwise you might hit a pull limit

Reply:
@kgaonkar6 thoughts on this?

Reply:
This could also go in a sbg repo, here's the dockerfile https://github.com/d3b-center/annotSV/blob/master/DockerFile